### PR TITLE
Fixed log statement in SusiService

### LINKED ISSUE #694: Fixed Log statements in SusiService
**Actual Behaviour**

The exception is having log level info. While in other files the log level is of Warning for Exception

**Expected Behaviour**

The log level should be of Exception.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes #694 

Changes: Changed log level to warning for exception.

### CODE REVIEW

**Comment by yashLadha** [comment]
Ping @saurabhjn76 @Orbiter @mariobehling

**Comment by yashLadha** [comment]
Ping @saurabhjn76 @Orbiter @mariobehling

File: src/ai/susi/server/api/susi/SusiService.java [hunk 1]
```diff
@@ -21,30 +21,14 @@
...
-import ai.susi.server.APIException;
-import ai.susi.server.APIHandler;
-import ai.susi.server.AbstractAPIHandler;
-import ai.susi.server.Authorization;
-import ai.susi.server.Query;
-import ai.susi.server.ServiceResponse;
-import ai.susi.server.UserRole;
-
+import ai.susi.mind.*;
+import ai.susi.server.*;
```
Comment (sudheesh001) [inline src/ai/susi/server/api/susi/SusiService.java]: Using wildcards for imports is a _BAD_ idea.

Comment (Orbiter) [reply src/ai/susi/server/api/susi/SusiService.java]: I agree completely. It's also bad that this is a mixed change together with the fix. But lets accept this by now to move on. Please @yashLadha do not use wild cards in the future.

**Comment by yashLadha** [comment]
Sure @Orbiter will take care in future.

**Comment by mariobehling** [comment]
@yashLadha Please resolve merge conflicts.

**Comment by yashLadha** [comment]
Working on it

**Comment by yashLadha** [comment]
Done @Orbiter . GTG

**Pull request merged**